CAP/gap/InstallAdds.gi

```diff
@@ -128,6 +128,19 @@ InstallGlobalFunction( CapInternalInstallAdd,
         
         InstallMethod( ValueGlobal( install_name ), replaced_filter_list{[ 2 .. Length( replaced_filter_list ) ]}, get_convenience_function( ValueGlobal( install_name ) ) );
         
+        if IsPackageMarkedForLoading( "JuliaInterface", ">= 0.2" ) then
```

Comment:
Semantically, this block should go outside of the enclosing `if` (even if technically `"list_of_objects" in filter_list or "list_of_morphisms" in filter_list` implies `record.install_convenience_without_category`).